fix(releaseNotes): change support email to UI reference (#18175)
newrelic/docs-website: src/content/docs/alerts/get-notified/notification-integrations.mdx

  ### Prerequisites [#slack-prereqs]

>   Your Slack workspace needs to have the [New Relic application](https://newrelic.slack.com/apps/AP92KQJS3-new-relic?tab=more_info)(or the [EU App](https://slack.com/apps/AS5D75HQQ-new-relic-eu?tab=more_info) for `one.eu.newrelic` customers) installed. A workspace administrator must approve the application before you can individually install it. If you need assistance, you can get help by following these steps:

Reviewer: How about? 

"If you need assistance, follow these steps in the UI:"

Author: Ah, thanks for the catch

>   Your Slack workspace needs to have the [New Relic application](https://newrelic.slack.com/apps/AP92KQJS3-new-relic?tab=more_info)(or the [EU App](https://slack.com/apps/AS5D75HQQ-new-relic-eu?tab=more_info) for `one.eu.newrelic` customers) installed. A workspace administrator must approve the application before you can individually install it. If you need assistance, you can get help by following these steps:

Author: ```suggestion
  Your Slack workspace needs to have the [New Relic application](https://newrelic.slack.com/apps/AP92KQJS3-new-relic?tab=more_info)(or the [EU App](https://slack.com/apps/AS5D75HQQ-new-relic-eu?tab=more_info) for `one.eu.newrelic` customers) installed. A workspace administrator must approve the application before you can individually install it. If you need assistance, follow these steps in the UI:
```

>     1. Log in to New Relic.

Reviewer: Not sure we need step 1, because they'll be prompted to log in when they go to that Help Center link.

Author: When you're prompted to log in, it doesn't direct back to that help center link so they'd have to come back anyway to the doc

Reviewer: That's too bad!
